dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/resources/docker/basic/docker-compose.yaml

```diff
@@ -26,12 +26,18 @@ services:
     ports:
       - "12345:12345"
     networks:
-      - api-test
+      network:
+        ipv4_address: 10.5.0.5
     healthcheck:
       test: [ "CMD", "curl", "http://localhost:12345/actuator/health" ]
       interval: 5s
       timeout: 60s
       retries: 120
 
 networks:
-  api-test:
+  network:
+    driver: bridge
+    ipam:
+      config:
+        - subnet: 10.5.0.0/16
+          gateway: 10.5.0.1
```

Comment:
Why do we need this?

Reply:
We need to know the worker ip address when calling `saveWorkerGroup`. If we use a random ip, it would figure out there is no such worker and lead to failure.
![image](https://user-images.githubusercontent.com/34905992/232653100-758968b7-43c3-489f-ab22-24ef05dfb665.png)